lms/templates/vert_module.html

```diff
@@ -7,7 +11,7 @@
 <div class="vert-mod">
 % for idx, item in enumerate(items):
   <div class="vert vert-${idx}" data-id="${item['id']}">
-    ${item['content'] | n, unicode}
+    ${HTML(item['content'])}
```

Comment:
I'm also interested in this change, it seems potentially dangerous (esp. in light of Andy's XSS POC) and I don't see what in the rest of the PR would necessitate this allowing HTML now.